Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -370,43 +370,63 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
         return String.localizedStringWithFormat(format, readCount, numArticlesString, numLanguagesString)
     }
     
-    func personalizedYouReadSlideTitleV3(readCount: Int, minutesRead: Int) -> String {
+    func personalizedYouReadSlideTitleV3(readCount: Int) -> String {
+        let minutesRead = readCount * 3
         let format = WMFLocalizedString("year-in-review-personalized-reading-title-v3-format", value: "You spent {{PLURAL:%1$d|%1$d minute|%1$d minutes}} reading {{PLURAL:%2$d|%2$d article|%2$d articles}} in 2025", comment: "Year in review, personalized reading article count slide title for users that read articles. %1$d is replaced with the number of minutes the user spent reading and %2$d is replaced with the number of articles the user read in 2025.")
         return String.localizedStringWithFormat(format, minutesRead, readCount)
     }
 
-    func personalizedYouReadSlideSubtitleV3(readCount: Int) -> String {
-        
-        let minReadCount = 1
-        let maxReadCount = 43740
-        let containedReadCount = min(max(readCount, minReadCount), maxReadCount)
-        let percentage = 100 - (Double(containedReadCount) / Double(maxReadCount) * 100)
-        
-        let percentageString: String
-        if readCount > maxReadCount {
-            percentageString = "0.001"
-        } else {
-            percentageString = String(format: "%.3f", percentage)
+    func percentileRange(for readCount: Int) -> String {
+        switch readCount {
```

Comment:
@l-olson1214  When I pass in a readCount of 336, I was expecting [50%](https://docs.google.com/spreadsheets/d/1fzdsQbC41Cw9bUH_n7CWx3O7WCLVs4dWaQQ_SXhROf8/edit?gid=542339239#gid=542339239) but it resolves to top 0.01 percent. Can you look into that?

Also to be safe, maybe we should have default and <= 335 return nil, and omit the first sentence if `percentileRange` evaluates to nil.